auth-api/migrations/versions/2023_09_05_1ddf3b37e17f_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 1ddf3b37e17f
+Revises: 98bee9969323
+Create Date: 2023-09-05 11:33:50.692272
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1ddf3b37e17f'
+down_revision = '98bee9969323'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('affiliations', sa.Column('environment', sa.String(length=20), nullable=False))
```

Comment:
Wont this have issues where environment is already empty?

Reply:
Ok, so merging this PR, it's going to explode and fail wont it? nullable should be set to True until you run the update script?